### DISCUSSION TITLE
Axolotl for Base Model Inference

### DISCUSSION THREAD

**Discussion by Maaalik** [axolotl-ai-cloud/axolotl#1538] [Q&A]
(Sorry, this is possibly a stupid question!) Could one use axolotl to gather the inferences of a "raw" model on a whole test set, saving these inferences in order to evaluate them at a later stage as a baseline to be compared to the output of finetuned variants? If so, what cli command should be used? What yaml template? Or is axolotl simply not made for this?

**Comment by NanoCode012** [comment]
Late update, I would recommend vllm for any future readers. Axolotl is more focused on training pipeline.
